docs/configuration.rst

```diff
@@ -19,7 +19,7 @@ With Tutor, all Open edX deployment parameters are stored in a single ``config.y
 
     cat "$(tutor config printroot)/config.yml"
 
-By default, this file contains only the required configuration parameters for running the platform. Optional configuration parameters may also be specified to modify the default behaviour. To do so, you can edit the ``config.yml`` file manually::
+By default, this file contains only the required configuration parameters for running the platform. Optional configuration parameters may also be specified to modify the default behavior. To do so, you can edit the ``config.yml`` file manually::
```

Comment:
Could we check the vocab and grammar with UK English? This is the convention that I try to adopt across Tutor. For instance, "behaviour" is British while "behavior" is US: https://www.wordreference.com/enfr/behaviour

Reply:
ok, I wasn't aware of the UK over US situation. behaviour It is! :)